### PR TITLE
Empty sentry bugfix

### PR DESCRIPTION
**Pull request by dsavchenko**
dispatcher was not working with empty sentry url in config

### CODE REVIEW

File: cdci_data_analysis/pytest_fixtures.py [hunk 1]
```diff
@@ -362,6 +362,36 @@ def dispatcher_test_conf_fn(tmpdir):
...
+        bind_port: 8001
+    email_options:
+        smtp_server: 'localhost'
+        sender_email_address: 'team@odahub.io'
+        cc_receivers_email_addresses: ['team@odahub.io']
+        bcc_receivers_email_addresses: ['teamBcc@odahub.io']
+        smtp_port: 61025
+        smtp_server_password: ''
+        email_sending_timeout: True
+        email_sending_timeout_default_threshold: 1800
```
Comment (burnout87) [inline cdci_data_analysis/pytest_fixtures.py:389]: This an be avoided by simply removing removing the `sentry_url` value from the previous one, or maybe with a parameter that controls the value of `sentry_url`. No need to duplicate.

**Review by burnout87 — CHANGES_REQUESTED** [review]
Perhaps we should not send any sentry in case the url is not specified.

Is it breaking specifically somewhere else?

**Comment by volodymyrss** [comment]
eh, why so many changes, not related to sentry?

**Review by volodymyrss — COMMENTED** [review]
how's that this one has diverged so much? could you re-create it maybe? (could be related to the force pushes)

**Comment by volodymyrss** [comment]
@dsavchenko I emulated that recreating the PR shows correct changes from just a few commits.

**Comment by dsavchenko** [comment]
Yes, let me recreate the PR.

**Comment by volodymyrss** [comment]
> Yes, let me recreate the PR.

dont forget the [comment](https://github.com/oda-hub/dispatcher-app/pull/283#pullrequestreview-812150475) though!

**Pull request closed without merging**